providers/implementations/rands/seed_src.c

```diff
@@ -170,6 +172,42 @@ static int seed_src_verify_zeroization(ossl_unused void *vseed)
     return 1;
 }
 
+static size_t seed_get_seed(void *vseed, unsigned char **pout,
+                            int entropy, size_t min_len, size_t max_len,
+                            int prediction_resistance,
+                            const unsigned char *adin, size_t adin_len)
+{
+    size_t bytes_needed;
+    unsigned char *p;
+
+    /* Figure out how many bytes we need */
+    bytes_needed = entropy >= 0 ? (entropy + 7) / 8 : 0;
```

Comment:
Is this assuming 8 bits of strength per byte??

Reply:
Yes.  For other seed sources, it need not be true but for ours it's a safe assumption.

Reply:
Did you push the comment for this?

Reply:
I have now.